views/publication/show.templ

```diff
@@ -9,157 +9,237 @@ import (
 templ Show(c *ctx.Ctx, p *models.Publication, redirectURL string) {
 	@views.PageLayout(c, c.Loc.Get("publication.page.show.title"), nil) {
 		<div class="w-100 u-scroll-wrapper">
-			<div class="bg-white" data-collapsed="true">
-				<div class="bc-navbar bc-navbar--large bc-navbar--white">
-					<div class="bc-toolbar">
-						<div class="bc-toolbar-left">
-							<div class="bc-toolbar-item">
-								<a class="btn btn-link btn-link-muted" href={ templ.URL(redirectURL) }>
-									<i class="if if-arrow-left"></i>
-									<div class="btn-text">Publications overview</div>
-								</a>
-							</div>
-						</div>
-						<div class="bc-toolbar-right">
-							if c.User.CanWithdrawPublication(p) {
-								<div class="bc-toolbar-item">
-									<button
-										class="btn btn-outline-danger"
-										hx-get={ c.PathTo("publication_confirm_withdraw", "id", p.ID, "redirect-url", c.CurrentURL.String()).String() }
-										hx-target="#modals"
-									>
-										<i class="if if-arrow-go-back"></i>
-										<span class="btn-text">Withdraw</span>
-									</button>
-								</div>
-							}
-							if c.User.CanPublishPublication(p) && p.Status == "returned" {
-								<div class="bc-toolbar-item">
-									<button
-										class="btn btn-success"
-										hx-get={ c.PathTo("publication_confirm_republish", "id", p.ID, "redirect-url", c.CurrentURL.String()).String() }
-										hx-target="#modals"
-									>
-										<div class="btn-text">Republish to Biblio</div>
-									</button>
-								</div>
-							}
-							if c.User.CanPublishPublication(p) &&  p.Status != "returned" {
-								<div class="bc-toolbar-item">
-									<button
-										class="btn btn-success"
-										hx-get={ c.PathTo("publication_confirm_publish", "id", p.ID, "redirect-url", c.CurrentURL.String()).String() }
-										hx-target="#modals"
-									>
-										<div class="btn-text">Publish to Biblio</div>
-									</button>
+			<div class="bg-white">
+				<div id="summary">
+					<div class="mx-6">
+						<div class="bc-toolbar bc-toolbar-md-responsive flex-column-reverse flex-md-row w-100">
+							<div class="bc-toolbar-left">
+								<div class="d-inline-flex align-items-center flex-wrap">
+									@views.BadgeStatus(p.Status)
+									if p.Locked {
+										<span class="c-subline text-nowrap me-3 pe-3 border-end" data-bs-toggle="tooltip" data-bs-title="Locked for editing" data-bs-placement="bottom">
+											<i class="if if-lock if--small if--muted"></i>
+											<span class="text-muted c-body-small ms-2">Locked</span>
+										</span>
+									}
+									<span class="c-subline text-nowrap me-3 pe-3 border-end">
+										{ c.Loc.Get("publication_types." + p.Type) }
+										if p.Classification != "" {
+											{ p.Classification }
+										}
+									</span>
```

Comment:
Publication type and classification are concatenated with just a space this way which feels odd to mee. I see the prototype doesn't include the classification so not sure if this is ok, but wanted to note that it doesn't look right anyway:

<img width="354" alt="image" src="https://github.com/user-attachments/assets/5cfafc2f-64c5-4829-abac-cb9a3f8121fa">